eng/Subsets.props

```diff
@@ -120,7 +120,14 @@
     <SubsetName Include="Mono.WasmRuntime" Description="The WebAssembly runtime." />
     <SubsetName Include="Mono.MsCorDbi" Description="The implementation of ICorDebug interface." />
     <SubsetName Include="Mono.Workloads" OnDemand="true" Description="Builds the installers and the insertion metadata for Blazor workloads." />
-    
+
+    <!-- Host -->
+    <SubsetName Include="Host" Description="The .NET hosts, packages, hosting libraries, and tests." />
+    <SubsetName Include="Host.Native" Description="The .NET hosts." />
+    <SubsetName Include="Host.Pkg" Description="The .NET host packages." />
+    <SubsetName Include="Host.Tools" Description="The .NET hosting libraries." />
+    <SubsetName Include="Host.Tests" Description="The .NET hosting tests." />
```

Comment:
Can you move the Host.Tests subset to after the libs subsets? The host tests consume the live libraries assets during test run, so they need to be sequenced after the libraries build.

I know the right place to comment this is lower, but I GitHub won't let me comment on the line.

Reply:
Yup, done.